exercises/beer-song/example.js

```diff
@@ -19,14 +19,14 @@ function action(currentVerse) {
   return `Take ${sbj} down and pass it around, `;
 }
 
-function nextBottle(currentVerse) {
-  return `${bottles(nextVerse(currentVerse)).toLowerCase()} of beer on the wall.\n`;
-}
-
 function nextVerse(currentVerse) {
   return currentVerse === 0 ? 99 : currentVerse - 1;
 }
 
+function nextBottle(currentVerse) {
+  return `${bottles(nextVerse(currentVerse)).toLowerCase()} of beer on the wall.\n`;
+}
+
```

Comment:
Moving this function is not required I think.

Reply:
It is because the nextVerse function needs to be defined before the nextBottle one.

Reply:
Was that an eslint error?

Reply:
`23:21  error  'nextVerse' was used before it was defined  no-use-before-define`

Something like this.

Reply:
Ahh ok 👍